include/hal/library/platform_lib.h

```diff
@@ -0,0 +1,33 @@
+/**
+    Copyright Notice:
+    Copyright 2022 DMTF. All rights reserved.
+    License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+**/
+
+#ifndef __PLATFORM_LIB_H__
+#define __PLATFORM_LIB_H__
+
+/**
+  Suspends the execution of the current thread until the time-out interval elapses.
+
+  @param[in] milliseconds     The time interval for which execution is to be suspended, in milliseconds.
+
+**/
+void libspdm_sleep(uint64_t milliseconds);
+
+/**
+  If no heartbeat arrives in microseconds, the watchdog timeout event
+  should terminate the session.
+
+  @param[in] microseconds     heartbeat period, in microseconds.
+
+**/
+boolean init_watchdog(uint64_t microseconds);
```

Comment:
I think we need rename it to start_watchdog(), and we need add stop_watchdog() and always call stop_watchdog() when the session is ended.